tabpfn_client/tests/unit/test_client.py

```diff
@@ -27,6 +28,14 @@ def test_try_connection_with_invalid_server(self, mock_server):
         mock_server.router.get(mock_server.endpoints.root.path).respond(404)
         self.assertFalse(self.client.try_connection())
 
+    @with_mock_server()
+    def test_try_connection_with_outdated_client(self, mock_server):
```

Comment:
In general, a better practice for the naming the test cases should include:
- test condition
- expected result

In this case, you could call it `def test_try_connection_with_outdated_client_raises_xxx`

ps. I figured that the old test cases were following this naming convention, but let's start applying this now :)